hyke/api/jobs_system.py

```diff
@@ -23,82 +23,40 @@
 
-    items = StatusEngine.objects.filter(Q(outcome=-1) & Q(formationtype__startswith="Hyke System"))
+    items = StatusEngine.objects.filter(
+        Q(outcome=StatusEngine.SCHEDULED) & Q(formationtype__startswith="Hyke System")
+    )
 
     print("Active items in the job: " + str(len(items)))
 
     db.close_old_connections()
 
-    for item in items:
-        if item.process == "Client Onboarding Survey" and item.processstate == 1 and item.outcome == -1:
-            try:
-                send_client_onboarding_survey(email=item.email)
-            except Exception as e:
-                logger.exception(f"Can't process Onboarding NPS Survey for status engine id={item.id}")
+    return items
 
-        elif item.process == "Payment error email" and item.processstate == 1 and item.outcome == -1:
-            send_transactional_email(
-                email=item.email, template="[Action required] - Please update your payment information",
-            )
-            print("[Action required] - Please update your payment information email is sent to " + item.email)
-        elif item.process == "Running flow" and item.processstate == 1 and item.outcome == -1:
-            ps = ProgressStatus.objects.get(email=item.email)
-            ps.bookkeepingsetupstatus = "completed"
-            ps.taxsetupstatus = "completed2"
-            ps.save()
-
-            StatusEngine.objects.get_or_create(
-                email=item.email,
-                process="Schedule Email",
-                formationtype="Hyke Daily",
-                processstate=1,
-                outcome=StatusEngine.SCHEDULED,
-                data="What's upcoming with Collective?",
-                defaults={"executed": timezone.now() + relativedelta(days=1)},
-            )
-
-            StatusEngine.objects.get_or_create(
-                email=item.email,
-                process="Running flow",
-                formationtype="Hyke System",
-                processstate=2,
-                defaults={"outcome": StatusEngine.SCHEDULED, "data": "---"},
-            )
-
-            schedule_onboarding_survey_sequence(email=item.email)
-            schedule_next_running_survey_sequence(email=item.email)
-
-            create_dropbox_folders(email=item.email)
 
-            print("Dropbox folders are created for " + item.email)
-
-            has_run_before = StatusEngine.objects.filter(
-                email=item.email, process=item.process, processstate=item.processstate, outcome=1,
-            ).exists()
-
-            if has_run_before:
-                print(
-                    "Not creating form w9 or emailing pops because dropbox folders job has already run for {}".format(
-                        item.email
-                    )
-                )
-        elif item.process == "Annual Report Uploaded" and item.outcome == -1:
+def scheduled_system(scheduled_items):
+    for item in scheduled_items:
+        if item.process == "Annual Report Uploaded":
```

Comment:
Anything in item.process (or any other string we are comparing against) should be a constant